packages/app/src/app/examples/send-ether/page.tsx

```diff
@@ -0,0 +1,118 @@
+'use client'
+import { useAccount, useBalance, useSendTransaction, useWaitForTransaction, usePrepareSendTransaction } from 'wagmi'
+import { useState, useEffect } from 'react'
+import { ethers } from 'ethers'
+import { parseEther } from 'viem'
+import { toast } from 'react-toastify'
+import 'react-toastify/dist/ReactToastify.css'
+
+import Ethereum from '../../../assets/icons/ethereum.png'
+
+export default function SendEther() {
+  const [to, setTo] = useState('')
+  const [isValidToAddress, setIsValidToAddress] = useState<boolean>(false)
+  const [amount, setAmount] = useState('0.01')
+
+  const { address } = useAccount()
+  const balance = useBalance({
+    address,
+  })
+
+  const { config, error: prepareError } = usePrepareSendTransaction({
+    to: isValidToAddress ? to : undefined,
+    value: parseEther(amount),
+  })
+
+  const { data, isLoading, sendTransaction } = useSendTransaction(config)
+
+  const {
+    isSuccess: txSuccess,
+    error: txError,
+    isLoading: txLoading,
+  } = useWaitForTransaction({
+    hash: data?.hash,
+  })
+
+  const handleSendTransation = () => {
+    if (prepareError) {
+      toast.error(`Transaction failed: ${prepareError.cause}`)
+      return
+    }
+    sendTransaction?.()
+  }
+
+  const handleToAdressInput = (to: string) => {
+    if (to.startsWith('0x')) setTo(to as `0x${string}`)
+    else setTo(`0x${to}`)
+    setIsValidToAddress(ethers.utils.isAddress(to))
+  }
+
+  useEffect(() => {
+    if (txSuccess) {
+      toast.success('Transaction successful')
+      balance.refetch()
+    } else if (txError) {
+      toast.error(`Transaction failed : ${txError.message}`)
+    }
+  }, [txSuccess, txError])
+
+  const formatBalance = (balance: string) => {
+    return Number.parseFloat(Number.parseFloat(balance).toFixed(2)).toExponential()
```

Comment:
Your balance comes in wei (1e18). You probably want to convert this different. 
e.g. Viem offers utils to parse/format units. See https://viem.sh/docs/utilities/formatEther

Reply:
No, wagmi returns the balance in Ether as a floating point number (the response is a string), for example, '0.123456789'. How do you prefer that I format the balance?

Reply:
Ok, I made changes, what do you think ?